lib/Manager.php

```diff
@@ -175,7 +175,6 @@ public function createRoomObject(array $row): Room {
 			(string) $row['password'],
 			(string) $row['remote_server'],
 			(string) $row['remote_token'],
-			(int) $row['active_guests'],
```

Comment:
This is a value on the model, please keep it injected and also keep the getter + setter to modify the value on the model (only the DB update part is basically moved to the RoomService